Servers/UI/OJS.Servers.Ui/ClientApp/src/common/constants.ts

```diff
@@ -15,9 +15,25 @@ enum ContestResultType {
     Full = 'full',
 }
 
+enum ContestErrors {
+    NotFound,
+    Expired,
+    NotRegistered,
+    NotEligible,
+}
+
+const ContestErrorMessages: Record<ContestErrors, string> = {
+    [ContestErrors.NotFound]: 'Contest not found!',
```

Comment:
Why is it needed to have these replicated on the frontend? Cant this be returned from the backend

Reply:
I removed them on the front end and now get the error message from the back end, which also simplified the code on the front end by a lot.